build/yamls/base/conf/antrea-agent.conf

```diff
@@ -116,3 +119,13 @@ featureGates:
 # whenever a Pod's container defines a specific port to be exposed (each container can define a list of ports as pod.spec.containers[].ports),
 # and all Node traffic directed to that port will be forwarded to the Pod.
 #nplPortRange: 40000-41000
+
+# The virtual IP for NodePort Service support. It must be a link-local IP otherwise the Agents will report error.
+#nodePortVirtualIP: 169.254.169.110
```

Comment:
Have you tried policy route approach? I did an experiment and it worked as we discussed. 
If NAT is not likely to be the final solution, better not to add this argument in case compatbility issue in future. We can just use a reserved IP in alpha phase I think.

Reply:
Why we care it must be link-local?

Reply:
@tnqn, I tried the policy route way.
If the destination is a loopback address we still need to do a DNAT. Considering this is an ALPHA feature for now, we can discuss it more in the future.

Reply:
@jianjuns, do you think we should use any forwardable address here?

Reply:
I want to understand the current design first. Why IPv4 must be loopback, and IPv6 must not be?

Reply:
Got more understanding after reading the design proposal. I think no need to restrict the IP must be link-local. We can recommend link-local though.

Reply:
Thanks @jianjuns, updated.

Reply:
I mean there are possibilities we don't need to do DNAT in future like the two approaches we have discussed. Even if we still need it for loopback address ( maybe we don't need it for this case as well), I think hardcoding a reserved IP is enough at this stage.
I just want to avoid introducing implementation specific configurations when it's not stable so we don't have the complexity of upgrade (If user set it and we delete it in future, the process will crash). 
And it doesn't seem that user can get what the configuration means and how to use it with the current comments.

Reply:
@tnqn, thanks for the explanation. I removed these two options and made them constant for now.